3529/2/Lib/Utiles/path_loader.rb

```diff
@@ -0,0 +1,15 @@
+require 'fileutils'
+
+class PathLoader
+  def initialize(user_id, chat_id)
+    @session_id = "#{user_id}:#{chat_id}"
```

Comment:
Mb better to use some static number? like rubizza numbers? Fell free to just resolve this 1